docs/examples/promise/07_make_transfer_with_allowed_block_permissions_only/index.js

```diff
@@ -0,0 +1,47 @@
+// @ts-check
+// Import the API, Keyring and some utility functions
+const { ApiPromise } = require('@polkadot/api');
+const { Keyring } = require('@polkadot/keyring');
+const { ExtrinsicEra } = require('@polkadot/types/type');
+
+
+
+const BOB = '5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty';
+
+async function main () {
+  // Instantiate the API
+  const api = await ApiPromise.create(); //remove provider
+
+  // Constuct the keying after the API (crypto has an async init)
+  const keyring = new Keyring({ type: 'sr25519' });
+
+  // Add alice to our keyring with a hard-deived path (empty phrase, so uses dev)
+  const alice = keyring.addFromUri('//Alice');
+
+  // Get nonce for account
+  const nonce = await api.query.system.accountNonce(alice.address());
```

Comment:
Since the newest `@polkadot/keyring`, this has been [changed](https://github.com/polkadot-js/common/blob/master/CHANGELOG.md#0931), so here it should be `alice.address`